templates/terraform/property_documentation.erb

```diff
@@ -14,7 +14,12 @@
 <%   end -%>
 <% end -%>
 <%= indent(property.description.strip.gsub("\n\n", "\n"), 2) %>
-<% if property.is_a?(Api::Type::Enum) && !property.output && !property.skip_docs_values -%>
+<% if property.is_a?(Api::Type::Array) && property.item_type.is_a?(Api::Type::Enum) && !property.output && !property.item_type.skip_docs_values -%>
```

Comment:
May need to add this to https://github.com/GoogleCloudPlatform/magic-modules/blob/master/templates/terraform/schema_property.erb as well. The same documentation appears there.

Reply:
Thanks, done.